test/functional/test_framework/test_framework.py

```diff
@@ -143,8 +143,8 @@ def main(self):
         config = configparser.ConfigParser()
         config.read_file(open(self.options.configfile))
         self.config = config
-        self.options.bitcoind = os.getenv("BITCOIND", default=config["environment"]["BUILDDIR"] + '/src/bitcoind' + config["environment"]["EXEEXT"])
-        self.options.bitcoincli = os.getenv("BITCOINCLI", default=config["environment"]["BUILDDIR"] + '/src/bitcoin-cli' + config["environment"]["EXEEXT"])
+        self.options.bitcoind = os.path.join(config["environment"]["BUILDDIR"], os.getenv("BITCOIND", default='src/bitcoind' + config["environment"]["EXEEXT"]))
+        self.options.bitcoincli = os.path.join(config["environment"]["BUILDDIR"], os.getenv("BITCOINCLI", default='src/bitcoin-cli' + config["environment"]["EXEEXT"]))
```

Comment:
Why is this changed?

We already export the PATH two lines below, so setting `BITCOIND=bitcoin-node` should be sufficient

Reply:
Oh, I didn't know about the PATH feature, so I'll revert this.

I changed this to get `BITCOIND=src/bitcoin-node test/functional/test_runner.py` not to fail, but I should probably take advantage of the PATH setting and change `BITCOIND=src/bitcoin-node` to `BITCOIND=bitcoin-node` instead.